libr/core/cmd_debug.inc.c

```diff
@@ -5194,13 +5194,13 @@ static int run_buffer_dxr(RCore *core, RBuffer *buf, bool print, bool ignore_sta
 	return ret;
 }
 
-// TODO: dd commands need tests in archos/linux-x64/cmd_dd
 // TODO: update the book page at src/debugger/files.html
 static int cmd_debug_desc(RCore *core, const char *input) {
 	int argc;
 	char **argv;
 	bool needs_live_process = false;
 	bool print = false; // enabled with *, print the command instead of running it
+	bool has_debugger = r_config_get_b (core->config, "cfg.debug");
```

Comment:
Rename variable to cfg_debug. Hasdebugger sounds more like a compile time feature